eessi_bot_software_layer.py

```diff
@@ -9,65 +9,81 @@
 #
```

Comment:
@trz42 Add yourself as an author to this script (and also add @bedroge while you're at it)